win32ss/user/ntuser/scrollbar.c

```diff
@@ -687,6 +687,11 @@ co_IntGetScrollBarInfo(PWND Window, LONG idObject, PSCROLLBARINFO psbi)
 
    sbi = IntGetScrollbarInfoFromWindow(Window, Bar);
    pSBData = IntGetSBData(Window, Bar);
+   if (pSBData == NULL)
```

Comment:
There are similar instances of a possible NULL check needed:
- in co_IntSetScrollInfo:
  https://github.com/reactos/reactos/blob/d809cd0f7f06d14c4b8816946675f98f5d047660/win32ss/user/ntuser/scrollbar.c#L496-L499
- in co_IntCreateScrollBars:
  https://github.com/reactos/reactos/blob/d809cd0f7f06d14c4b8816946675f98f5d047660/win32ss/user/ntuser/scrollbar.c#L789-L792

Reply:
Fixed the first one. The second one doesn't need it, because it doesn't use `SB_CTL`.